judo-runtime-core-export/src/main/java/hu/blackbelt/judo/runtime/core/export/JxlExportUtil.java

```diff
@@ -0,0 +1,205 @@
+                XlsArea dataArea = new XlsArea(sheetName + "!A2:" + column + "2", transformer);
+
+                EachCommand employeeEachCommand = new EachCommand("context", "list", dataArea);
+                headerArea.addCommand("A2:" + column +"2", employeeEachCommand);
+
+                Context context = new Context();
+                context.putVar("list", transformPayloadList(list));
+
+                // To debug use: headerArea.applyAt(new CellRef(RESULT_SHEET_NAME + "!A1"), context)
+                headerArea.applyAt(new CellRef(sheetName + "!A1"), context);
+                transformer.write();
+        }
+    }
+
+    public static InputStream createExcelExportToInputStream(String sheetName, OutputStream outputStream, List<Payload> list, Map<String, Class<?>> targetTypes, List<String> attributes) throws IOException {
+        Workbook workbook = new XSSFWorkbook();
+        createTemplateSheet(workbook, sheetName, targetTypes, attributes);
+
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+
+        if (targetTypes.size() > 0) {
+            char column = ((char) ((int) 'A' + targetTypes.size()));
+
+            SelectSheetsForStreamingPoiTransformer transformer = new SelectSheetsForStreamingPoiTransformer(workbook);
+            transformer.setOutputStream(byteArrayOutputStream);
+
+            XlsArea headerArea = new XlsArea(sheetName + "!A1:" + column + list.size() + 1, transformer);
+            XlsArea dataArea = new XlsArea(sheetName + "!A2:" + column + "2", transformer);
+
+            EachCommand employeeEachCommand = new EachCommand("context", "list", dataArea);
+            headerArea.addCommand("A2:" + column +"2", employeeEachCommand);
+
+            Context context = new Context();
+            context.putVar("list", transformPayloadList(list));
+
+            // To debug use: headerArea.applyAt(new CellRef(RESULT_SHEET_NAME + "!A1"), context)
+            headerArea.applyAt(new CellRef(sheetName + "!A1"), context);
+            transformer.write();
+        }
+
+        if (outputStream != null) {
+            outputStream.write(byteArrayOutputStream.toByteArray());
+        }
+
+        return convertOutputStreamIntoInputStream(byteArrayOutputStream);
+    }
+
+
+    private static void createTemplateSheet(Workbook workbook, String templateSheetName, Map<String, Class<?>> targetTypes, List<String> attributes) throws IOException {
+        workbook.createSheet(templateSheetName);
+        workbook.getSheet(templateSheetName).createRow(0);
+        workbook.getSheet(templateSheetName).createRow(1);
+
+        int column = 0;
+
+        CreationHelper createHelper = workbook.getCreationHelper();
+        CellStyle timeCellStyle = workbook.createCellStyle();
+        timeCellStyle.setDataFormat(
+                createHelper.createDataFormat().getFormat("hh:mm:ss"));
+        CellStyle dateCellStyle = workbook.createCellStyle();
+        dateCellStyle.setDataFormat(
```

Comment:
Is there no built-in formatter for this? We cannot fixate date/time formats, and if this is somehow "the standard" format for excel, then there must be some built-in helper.

Reply:
I did not find built in formatter for this.